proto/cosmos/crypto/multisig/keys.proto

```diff
@@ -22,8 +22,8 @@ message LegacyAminoPubKey {
   // by default marshal uint32 as a number).
   // 2. The `public_keys` field is renamed to `pubkeys`, which is also
   // reflected in the `legacy_amino_field_name` annotation.
-  option (legacy_amino.message_encoding)     = "threshold always as string";
-  option (gogoproto.goproto_getters) = false;
+  option (legacy_amino.message_encoding) = "threshold always as string";
```

Comment:
Would it make sense to have a soft convention of using underscores, no spaces for these? Maybe for this one something like this:

```
option (legacy_amino.message_encoding) = "threshold_string";
```

Reply:
fixed in e4949c1dfa24beacaf6039bc2497cfc2d20b1ef5